openstack/tools/charmed_openstack_functest_runner.sh

```diff
@@ -133,14 +133,10 @@ FIP_MIN_ABC=${FIP_MIN%.*}
 FIP_MIN_D=${FIP_MIN##*.}
 FIP_MIN=${FIP_MIN_ABC}.$(($FIP_MIN_D + 64))
 
-CIDR_OAM=$(openstack subnet show subnet_${OS_USERNAME}-psd -c cidr -f value)
-OAM_MAX=$(ipcalc $CIDR_OAM| awk '$1=="HostMax:" {print $2}')
-OAM_MIN=$(ipcalc $CIDR_OAM| awk '$1=="HostMin:" {print $2}')
-OAM_MIN_ABC=${OAM_MIN%.*}
-OAM_MAX_D=${OAM_MAX##*.}
-# Picking last two addresses and hoping they dont get used by Neutron.
-export {OS,TEST}_VIP00=${OAM_MIN_ABC}.$(($OAM_MAX_D - 1))
-export {OS,TEST}_VIP01=${OAM_MIN_ABC}.$(($OAM_MAX_D - 2))
+# Setup vips needed by zaza tests.
+for ((i=2;i;i-=1)); do
+    export {OS,TEST}_VIP0$((i-1))=$(create_zaza_vip (($i-1)))
+done
```

Comment:
/home/ubuntu/stsstack-bundles/openstack/tools/charmed_openstack_functest_runner.sh: command substitution: line 139: syntax error near unexpected token `('

Reply:
omg

Reply:
why didnt bashate find that

Reply:
anyway, fixed.